webui/react/.gitignore

```diff
@@ -9,7 +9,6 @@
 
 # production
 build/
-build-storybook/
```

Comment:
nit: I think this should probably stay in the gitignore so that no one accidentally commits an old build-storybook folder